tidy3d/web/auth.py

```diff
@@ -128,4 +123,41 @@ def get_credentials() -> None:
             return
 
         # otherwise, prompt again
-        log.info(f"Unknown response: {keep_logged_in}")
+        log.info(f"Unknown response: {keep_logged_in}.")
+
+
+def __get_credential_from_stored_file():
+    with open(credential_path, "r", encoding="utf-8") as fp:
+        auth_json = json.load(fp)
+    email = auth_json["email"]
+    password = auth_json["password"]
+    return email, password
+
+
+def __get_credential_from_env():
```

Comment:
Could you add docstrings to these functions? just to describe what the logic is doing. For example:
```py
def _get_credential_from_env():
    """Try to get email and password from environment variables, if not encoded, encode it."""
```
or something similar, just to help us know what these functions should do, thanks